fast_dp/fast_dp.py

```diff
@@ -130,48 +127,54 @@ def set_start_image(self, start_image):
 
         assert(self._start_image is None)
 
-        # check input is image file
+        # check input is image file, and exists
+        if not os.path.exists(start_image):
+            raise RuntimeError('%s does not exist' % start_image)
         if not os.path.isfile(start_image):
-            raise RuntimeError('no image provided: data collection cancelled?')
+            raise RuntimeError('%s is not a file' % start_image)
```

Comment:
```suggestion
            raise ValueError('%s is not a file' % start_image)
```
is possibly the slightly more accurate exception type.
May need to check if this is supposed to be caught upstream as ```RuntimeError```, because ```ValueError``` is not a subclass of ```RuntimeError```.